mod/wikirate_csv_import/set/self/import_tool.rb

```diff
@@ -21,13 +21,19 @@ def tab_content key
       "<h5 class='mt-3'>New import file</h5>",
       new_import_form(codename),
       "<h5 class='mt-5'>Recent imports</h5>",
-      list_group(recent_imports_list(codename))
+      list_group(recent_imports_list(codename)),
+      "<h5 class='mt-5'>All import files</h5>",
+      list_group(import_file_cards(codename))
     ]
   end
 
   def new_import_form import_type
-    nest(Card.new(type: import_type),
-         view: :new, hide: [:header, :menu, :new_type_formgroup])
+    if ok? :create
+      nest(Card.new(type: import_type),
+           view: :new, hide: [:header, :menu, :new_type_formgroup])
+    else
+      "You have to #{signin_link} to create new import files."
+    end
```

Comment:
doesn't the :new view already handle this with perms: :create ?

Reply:
...oh, it prompts to add.  This is part of the unknown API I want to improve...

Reply:
no, it's just blank